pkg/aspect/test/test_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"aspect.build/cli/pkg/ioutils"
 	"github.com/golang/mock/gomock"
 	. "github.com/onsi/gomega"
+	bep_mock "aspect.build/cli/pkg/plugin/system/bep/mock"
```

Comment:
Could you reorder the import statements here to separate the third-party from the first-party packages, please?

Reply:
Done. I am assuming golang has some linters available. Can we set one up to catch issues like this?

Reply:
I don't think it's needed. The auto importer and formatter will detect our logic and keep them correct going forward.

Reply:
Sounds good